sensor/common/registry/registry_store.go

```diff
@@ -56,18 +65,21 @@ type CheckTLS func(ctx context.Context, origAddr string) (bool, error)
 // The passed-in CheckTLS is used to check if a registry uses TLS.
 // If checkTLS is nil, tlscheck.CheckTLS is used by default.
 func NewRegistryStore(checkTLS CheckTLS) *Store {
-	factory := registries.NewFactory(registries.FactoryOptions{
+	allRegsFactory := registries.NewFactory(registries.FactoryOptions{})
+
+	someRegsFactory := registries.NewFactory(registries.FactoryOptions{
```

Comment:
what is all vs some?

Reply:
All includes all the integration types that are known, some includes the few types that the registry store currently creates based on the pull secret endpoint (ie: rhel or not-rhel). I debated on an appropriate name instead of `some`, like `known`, `possible`, or `rhelDockerFactory`.  None of them felt 'great' so I used `some`. Have a suggestion?

Reply:
Thinking about this more, i'm going to remove the second factory and just have the one. Initially my intent was to have separate factories one to mirror 'central' (ie: with repoList) for synced integrations and the other specific to the internals of the registry store + pull secrets. I will not be using repoList for either integrations so makes sense to combine the factories.